ONSdigital/eq-author-app
# EAR-1908 accessibility issue in comments MVP

### What is the context of this PR?

Ticket: https://jira.ons.gov.uk/browse/EAR-1908

Fixes accessibility issues in the comment section and improves contrast.

### How to review

- Open the comment section in your local author
  - then, ensure all the confirmation buttons have the correct text and background color
  - add a comment and when adding a reply ensure the cancel button has the correct color scheme
  - launch the wave evaluation tool and ensure there are no contrast errors
- Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
